### ISSUE TITLE
Website not visible 

### ISSUE THREAD

**Issue by Bisu172** [Th30neAnd0nly/TrackDown#3]
The link of input website is not visible in output link

**Comment by prateek-chaubey** [comment]
Can you elaborate it more.

**Comment by Bisu172** [comment]
I agree that whoever click my link I'll get his pic, location etc...
But the link that bot provide me doesn't work nothing is visible in the
link ...
